server/pgClient.js

```diff
@@ -0,0 +1,77 @@
+const client = require('../database/pgConnect.js');
+
+exports.getAll = (req, res) => {
+  let query = `SELECT * FROM records WHERE id=${req.params.id}`;
+  client.query(query, (error, results) => {
+    if (error) {
+      return console.log('error happened', error);
+    }
+    var result = [{
+      packaging: {
+        measurments: {
```

Comment:
measurements is missing and e. I like how you did this to get the camel casing back before it goes to front end!

Reply:
@claire-melbourne yeah, the front-end is expecting it spelled like that, so I had to keep it the same. LOL. I'm trying not to mess with the front end at all right now, but I think eventually I will go and fix the spelling errors cause there are a few that I caught but couldn't change.